kafka-quickstart/src/test/java/org/acme/kafka/KafkaResource.java

```diff
@@ -9,19 +9,21 @@
 
 public class KafkaResource implements QuarkusTestResourceLifecycleManager {
 
-    private final KafkaContainer KAFKA = new KafkaContainer();
+    private KafkaContainer kafka;
 
     @Override
     public Map<String, String> start() {
-        KafkaContainer KAFKA = new KafkaContainer();
-        KAFKA.start();
-        System.setProperty("kafka.bootstrap.servers", KAFKA.getBootstrapServers());
+        kafka = new KafkaContainer();
+        kafka.start();
+        System.setProperty("kafka.bootstrap.servers", kafka.getBootstrapServers());
```

Comment:
Not sure what you're saying here?

Reply:
Instead of setting the system property, you can do the following:

```
return Collections.singletonMap("kafka.boostrap.servers", kafka.getBootstrapServers());
```